src/render/tutorials/getting-started-with-docker/2_0.html.md

```diff
@@ -315,9 +31,9 @@ project(":hello-module").projectDir = file("modules/hello-module")
 docker.image.name=knotx-example/knotx-docker-tutorial
 ```
 
-### Run
+## Build & Run
 
-Now we are ready to run. First, let's build your docker image:
+First, let's build your docker image:
```

Comment:
Can you please add some notes about:
- how Docker is integrated with Gradle build (separate `docker.gradle.kts`)
- apply changes to healthcheck and describe its purpose
- add some section for debugging linking to https://github.com/Knotx/knotx-starter-kit#docker-debugging
- dev setup (this section can be added to starter kit and here should be only some link)

Reply:
`docker.gradle.kts`, debugging - Done ✔️ 
healthcheck - it is covered in the `Tests` section
dev setup - it will be covered in the future (using docker volumes)